dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/signatures/KotlinSignatureProvider.kt

```diff
@@ -413,7 +413,7 @@ public class KotlinSignatureProvider(
                 processExtraModifiers(t)
                 keyword("typealias ")
                 group(styles = mainStyles + t.stylesIfDeprecated(sourceSet)) {
-                    signatureForProjection(t.type)
+                    signatureForProjection(t.type, linkText = t.name)
```

Comment:
TBH, this feels a bit like a hack (hope that it will not backfire on us)
Maybe, it will be better to call this parameter somehow `typealias` related, as it's really safe to use, only when we know that `p` will be a `GenericTypeConstructor` - as in `DTypealias`.

Or we can even not use `signatureForProjection` here, as it feels like an abuse, as it's not really a projection here, but a declaration itself (IMO). In that case, it might be possible to copy or reuse the logic that is used for other class-like declarations [here](https://github.com/Kotlin/dokka/blob/e76721618b9c12c84154b16d680aa2c800c00659/dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/signatures/KotlinSignatureProvider.kt#L188-L196).

Still, feel free to ignore. I'm not insisting here.